docs/user-guide/walkthrough/k8s201.md

```diff
@@ -9,114 +9,143 @@ scaling.
+kubectl get pods -l app=nginx
+```
+
 For more information, see [Labels](/docs/user-guide/labels).
-They are a core concept used by two additional Kubernetes building blocks: Replication Controllers and Services.
+They are a core concept used by two additional Kubernetes building blocks: Deployments and Services.
+
+
+## Deployments
+
+OK, now you know how to make awesome, multi-container, labeled pods and you want to use them to build an application, you might be tempted to just start building a whole bunch of individual pods, but if you do that, a whole host of operational concerns pop up.  For example: how will you scale the number of pods up or down?  how will you rollout a new release?
+
+The answer to those questions and more is to use a [_Deployment_](/docs/user-guide/deployment.md) to manage maintaining and updating your running _Pods_.
+
+A deployment object defines a pod creation template (a "cookie-cutter" if you will) and desired replica count.  The deployment uses a label selector to identify the pods it manages, and will create or delete pods as needed to meet the replica count.  Deployments are also used to manage safely rolling out changes to your running pods.
+
+Here is a deployment that instantiates two nginx pods:
+
+{% include code.html language="yaml" file="deployment.yaml" ghlink="/docs/user-guide/walkthrough/deployment.yaml" %}
+
 
+#### Deployment Management
 
-## Replication Controllers
+Create an nginx deployment ([deployment.yaml](/docs/user-guide/walkthrough/deployment.yaml)):
 
-OK, now you know how to make awesome, multi-container, labeled pods and you want to use them to build an application, you might be tempted to just start building a whole bunch of individual pods, but if you do that, a whole host of operational concerns pop up.  For example: how will you scale the number of pods up or down and how will you ensure that all pods are homogeneous?
+Copy this to a loca file named `deployment.yaml`
 
-Replication controllers are the objects to answer these questions.  A replication controller combines a template for pod creation (a "cookie-cutter" if you will) and a number of desired replicas, into a single Kubernetes object.  The replication controller also contains a label selector that identifies the set of objects managed by the replication controller.  The replication controller constantly measures the size of this set relative to the desired size, and takes action by creating or deleting pods.
+```shell
+kubectl create -f ./deployment.yaml
+```
 
-For example, here is a replication controller that instantiates two nginx pods ([replication-controller.yaml](/docs/user-guide/walkthrough/replication-controller.yaml)):
-
-{% include code.html language="yaml" file="replication-controller.yaml" ghlink="/docs/user-guide/walkthrough/replication-controller.yaml" %}
+List all deployments:
 
+```shell
+kubectl get deployment
+```
 
-#### Replication Controller Management
+List the pods created by the deployment:

```

Comment:
The shell block isn't rendered well on the website. Perhaps adding a new line can solve this?
http://pwittrock.github.io/docs/user-guide/walkthrough/k8s201/#deployment-management
List the pods created by the deployment: `shell kubectl get pods -l app=nginx`